tasks/configure_master.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/bash
 
 declare PT__installdir
+# shellcheck disable=SC1090
 source "$PT__installdir/ca_extend/files/common.sh"
+PUPPET_BIN='/opt/puppetlabs/puppet/bin'
 
-\cp -R /etc/puppetlabs/puppet/ssl /var/puppetlabs/backups || fail "Error backing up '/etc/puppetlabs/puppet/ssl'"
+mkdir -p /var/puppetlabs/backups/
```

Comment:
We need to ensure security on the backup of the `ssl` directory. Can we do `cp -a` or chmod 700 on the dir, etc?

Reply:
`chmod` is a good idea.

Reply:
Oh I see, I thought you meant the `backups` directory itself.  We can do `cp -a` to preserve the `ssl` permissions.

Reply:
Done.